bluetooth/src/androidLibMain/kotlin/device/DefaultDeviceConnectionManager.kt

```diff
@@ -97,14 +97,15 @@ internal actual class DefaultDeviceConnectionManager(
 
         override fun onCharacteristicWrite(gatt: BluetoothGatt?, characteristic: BluetoothGattCharacteristic?, status: Int) {
             characteristic ?: return
-            if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.TIRAMISU) {
+            // TODO this seems to not work for Android TIRAMISU
+            /*if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.TIRAMISU) {
                 if (ActivityCompat.checkSelfPermission(context, Manifest.permission.BLUETOOTH_CONNECT) == PackageManager.PERMISSION_GRANTED && status == GATT_SUCCESS) {
                     gatt?.readCharacteristic(characteristic)
```

Comment:
To read from characteristic we have to check properties if the device support reading

Reply:
I think the same rule applied for the write. We have similar approach when we set notifications on/off (we check if properties contains this flag aka this characteristic supports notifications)